### ISSUE TITLE
Refactoring code

### ISSUE THREAD

**Issue by zhengj2007** [VEuPathDB-ontology/VEuPathDB-ontology#488]
Develop the ontology fully using templates for correcting warning message reported by ROBOT and facilitate the ontology development more efficiently.

**Comment by johnwjudkins** [comment]
@zhengj2007 Do we still need dc:Source and inSubset annotations? If you're not sure, I can keep them in eupath_terms for now, and we can discuss them later.

**Comment by zhengj2007** [comment]
I don't think we are going to use dc:Source and inSubset annotations in the ontology. We can consider to merge them to other annotations, such as definition source, ontology term requester.

**Comment by johnwjudkins** [comment]
Updated pull request #490.
I made the following annotation changes to simplify the template:
Source, inSubset annotation -> definition source
rdfs:comment -> editor note (Most term comments/notes are under 'editor note', and I thought they should be in only one column.)
There are two dc:contributor annotation properties. I moved values from both of them so they would all be values of http://purl.org/dc/elements/1.1/contributor.

Since terms are supposed to have only one parent, we should figure out what to do about the ICE terms that are a second parent to some terms. These ICE terms are:
[information on observation details](http://purl.obolibrary.org/obo/EUPATH_0000310)
[specimen information](http://purl.obolibrary.org/obo/EUPATH_0000609)
[treatment information](http://purl.obolibrary.org/obo/EUPATH_0000794)

**Comment by zhengj2007** [comment]
yes
